dags/oup/parser.py

```diff
@@ -136,3 +141,27 @@ def _get_authors(self, article: ET.Element):
                     }
                 )
         return authors
+
+    def _get_journal_title(self, article: ET.Element):
+        journal_title = get_text_value(
+            article.find(
+                "front/journal-meta/journal-title-group/journal-title",
+            )
+        )
+        if journal_title:
+            return journal_title
+        journal_publisher = get_text_value(
+            article.find(
+                "front/journal-meta/journal-title-group/abbrev-journal-title/[@abbrev-type='pubmed']"
+            )
+        )
+        if journal_publisher:
+            return journal_publisher
+
+        journal_publisher = get_text_value(
+            article.find(
+                "front/journal-meta/journal-title-group/abbrev-journal-title/[@abbrev-type='publisher']"
+            )
+        )
+        if journal_publisher:
+            return journal_publisher
```

Comment:
you can do `journal_publisher  = article.find(<first path>) or article.find(<second path>)` -> if the first condition is met second is not checked and then just return `journal_publisher` since if you don't return anything you return `None`

Reply:
It's actually not publisher, it's journal_title_short_form, I will change the name of the var